test/extensions/quic_listeners/quiche/active_quic_listener_test.cc

```diff
@@ -53,7 +53,7 @@ class ActiveQuicListenerTest : public testing::TestWithParam<Network::Address::I
           read_filter_->callbacks_->connection().addConnectionCallbacks(
               network_connection_callbacks_);
         }}),
-        connection_handler_(ENVOY_LOGGER(), *dispatcher_) {
```

Comment:
nits: the inheritance from `protected Logger::Loggable<Logger::Id::main>` can be removed.